apps/web/src/app/challenge/_components/comments/getCommentRouteData.ts

```diff
@@ -71,8 +83,6 @@ export async function getPaginatedComments({
 
   const totalPages = Math.ceil(totalComments / PAGESIZE);
 
-  const totalReplies = comments.reduce((a, c) => a + c._count.replies, 0);
-
   return {
     totalComments: totalReplies + totalComments,
```

Comment:
I'm hoping this doesn't break pagination, trust that you tested it but if it's broken we can always fix it